SeriesMaster-Frontend/App/Screens/MyShowsScreen.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import styled from "styled-components/native";
+import List from "../components/List";
+import ListItem from "../components/ListItem";
+import ListItemImage from "../components/ListItemImage";
+import ListItemText from "../components/ListItemText";
+import Tab from "../components/Tab";
+import colors from "../config/colors";
+import PropTypes from "prop-types";
+import imageSrc from "../assets/225030.jpg";
+
+const Container = styled.View`
+  flex: 1;
+  align-items: center;
+  background: ${colors.dark};
+`;
+
+const StyledView = styled.View`
+  flex-direction: row;
+  width: 100%;
+`;
+
+const MyShowsScreen = ({ navigation }) => {
+  const [watchlistIsActive, setWatchlistIsActive] = useState(false);
+  const [completedIsActive, setCompletedIsActive] = useState(true);
+
+  function handleSetWatchlist() {
+    setWatchlistIsActive(true);
+    setCompletedIsActive(false);
+  }
+  function handleSetCompleted() {
+    setWatchlistIsActive(false);
+    setCompletedIsActive(true);
+  }
```

Comment:
You could simplify this to a single state if you use a string or number.
Imagine, that you add a third tab. This leads to a third state and more complex `handleSet...` functions.

```suggestion
  const [activeTab, setActiveTab] = useState("watchlist");
```

The Tab could look like this:

```
      <Tab
          title="Watchlist"
          onPress={() => setActiveTab('watchlist')}
          isActive={activeTab === 'watchlist'}
        />
```

A different solution is to use a number instead of a string to identify the active tab.

Reply:
I wasn't really satisfied with my solution either, but couldn't think of a better one. This suggestion really helps a lot!